lib/version.js

```diff
@@ -18,11 +18,13 @@ function latest(versions) {
     return (/[0-9]/).test(version);
   });
   try {
-    version = semver.maxSatisfying(versions, '');
+    version = semver.maxSatisfying(versions, '', true) || versions.sort((a, b) => {
```

Comment:
Could we write these calls like `maxSatisfying(versions, /* range */ '', /* loose */ true)` to make it clearer what these parameters are? Same for the other uses of the `loose` parameter.

Reply:
Sounds good!

Reply:
Have updated to include the `/* loose */ true` comments, but it does look a little messy.
Do you think it would be better to do something like:
```javascript
const semverLoose = true;
version = maxSatisfying(versions, /* range */ '', semverLoose);
```
```javascript
const loose = true;
const range = '';
version = maxSatisfying(versions, range, loose);
```